### ISSUE TITLE
Following Quick Start docs. Cant resolve theme

### ISSUE THREAD

**Issue by msandora** [microsoft/fluentui#13968]
Hello Fluent team.

Following quick start guide https://fluentsite.z22.web.core.windows.net/quick-start
Immediately ran into a problem. I hope you can help.

I Created an app
> npx create-react-app my-app
installed fluentUi
> npm install @fluentui/react
installed northstar
> npm install @fluentui/react-northstar

When I run the app I am getting the following error:
./src/App.js
  Line 6:22:  'teamsTheme' is not defined  no-undef

Below is my package.json
```
{
  "name": "northstar-sandbox",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@fluentui/react": "^7.121.11",
    "@fluentui/react-northstar": "^0.50.0",
    "@testing-library/jest-dom": "^4.2.4",
    "@testing-library/react": "^9.5.0",
    "@testing-library/user-event": "^7.2.1",
    "react": "^16.13.1",
    "react-dom": "^16.13.1",
    "react-scripts": "3.4.1"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": "react-app"
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
}

```
Below is my component:
```
import React from 'react';
import { Provider, themes } from '@fluentui/react-northstar'
import { Button } from '@fluentui/react-northstar'

const App = () => {
  return (
    <Provider theme={teamsTheme}>
      <Button content="Get started" icon="play" iconPosition="after" primary />
    </Provider>
  );
}

export default App;
```

It surprises me that the 'themes' import is never used and that the docs indicate that 'teamsTheme' defines the theme. I feel like a piece of the puzzle is missing.

I am trying to create a simple fluent ui app using northstar themes. I would like to eventually have a select theme option that will toggle between the northstar themes. Are there any examples of an app like this out there for reference?

**Comment by jurokapsiar** [comment]
Thanks for reporting and sorry about the inconvenience. Looks like we have forgotten to fix the quick start guide after we restructured the package for tree shaking purposes.

Please use following to import Provider and theme:
```
import { Provider, teamsTheme } from '@fluentui/react-northstar'
```
Then you can use it as the quick start guide says:
```
  <Provider theme={teamsTheme}>
```
Here is a codesandbox that shows this: https://codesandbox.io/s/epic-faraday-6tyyi

**Comment by layershifter** [comment]
It was fixed in #13789, but was not released yet.

**Comment by msandora** [comment]
@jurokapsiar Thank you for your reply. I did realize the problem later that day. The documentation seems imprecise in some cases and I do wish there were more examples of how to use northstar components. 

I have been working with northstar for a week now and one thing that I am having a hard time with is getting the theme to work with standard fluent ui components.

If you can offer some guidance here and an example that would be great.
Again, just to be clear... I would like to use the teamsTheme color palette on a standard @fluentui/react component. Is that possible? 

Also I see in documentation that there is a virtualized table component but there is no example. https://fluentsite.z22.web.core.windows.net/virtualized-table I would like to build a detailslist component like this with fixed column headers. Are there any examples of how to do this?

**Comment by jurokapsiar** [comment]
fixed in #13789, released in 0.51.

@msandora  It is currently not possible to use teamsTheme in @fluentui/react. It is only available for @fluentui/react-northstar. We are figuring out a way to share common theme provider, but there are no concrete dates set for that.

If you find any problems with the documentation, please create issues so that we can fix them.

Source code for virtualized-table is available in the repo - please see packages/fluentui/docs/src/prototypes/VirtualizedTable/index.tsx.

Closing the issue as it has been resolved. Please follow up in new issues, if there are still problems or questions.
